SPHINXsys/src/shared/materials/inelastic_solid.h

```diff
@@ -38,15 +38,15 @@ namespace SPH {
 	* @class PlasticSolid
 	* @brief Abstract class for a generalized plastic solid
 	*/
-	class PlasticSolid : public LinearElasticSolid
+	class PlasticSolid : public NeoHookeanSolid
```

Comment:
why from NeoHookean?

Reply:
I think that plastic material derived from Neohookean better. Because plastic usually mean large deformation and large deformation, practically, can not be linear.

Reply:
okey, understand